src/releaseFiles/manifest/manifest.schema.json

```diff
@@ -42,23 +43,46 @@
       "description": "Type of this DAppNode Package. It is used to trigger some special features such as core functionality.",
       "default": "service",
       "examples": ["service", "dncore"],
-      "enum": ["service", "library", "dncore"],
-      "enumDescriptions": {
-        "service": "Can have ENVs a depend on library DAppNode Packages",
-        "library": "Should not have ENVs and cannot depend on service DAppNode Packages",
-        "dncore": "DAppNode Packages that are part of the DAppNode core and have special permissions"
-      }
+      "anyOf": [
```

Comment:
What the issue with the `enum` syntax? The docs will probably break as they only understand the `enum` syntax. Can you double check they can render the `anyOf` as pretty as `enum`?

Reply:
this is not a change from this PR. The manifest and wizard schemas exist already, they were just replaced. The new one is the compose schema which is the official one

The docs are not broken and understand the syntax
https://docs.dappnode.io/developers/manifest-reference#chain